docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+    )
+else:
+    merged_single_cells = "./BR00117012.parquet"
+
+# read only the metadata from parquet file
+parquet.ParquetFile(merged_single_cells).metadata
+
+# + editable=true slideshow={"slide_type": ""}
+# show the first few columns for metadata column names
+schema = parquet.read_schema(merged_single_cells)
+print("\n".join(str(schema).split("\n")[:12]))
+
+# +
+# set a list of metadata columns for use throughout
+metadata_cols = [
+    "Metadata_ImageNumber",
+    "Image_Metadata_Row",
+    "Image_Metadata_Site",
+    "Metadata_ObjectNumber",
+    "Metadata_Plate",
+    "Metadata_Well",
+    "Image_TableNumber",
+]
+
+# read only metadata columns with outlier-specific columns
+df_merged_single_cells = pd.read_parquet(
+    path=merged_single_cells,
+    columns=[
+        *metadata_cols,
+        "Nuclei_AreaShape_Area",
+        "Nuclei_AreaShape_FormFactor",
+        "Nuclei_AreaShape_Eccentricity",
+    ],
+)
+df_merged_single_cells.head()
+# -
+
+# label outliers within the dataset
+print("Large nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="large_nuclei",
+)
+
+# label outliers within the dataset
+print("Elongated nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="elongated_nuclei",
+)
+
+# label outliers within the dataset
+print("Small and low formfactor nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="small_and_low_formfactor_nuclei",
+)
+
+# label outliers within the dataset
+df_labeled_outliers = cosmicqc.analyze.label_outliers(
+    df=df_merged_single_cells,
+    include_threshold_scores=True,
+)
+# show added columns
+df_labeled_outliers[
+    [col for col in df_labeled_outliers.columns.tolist() if "cqc." in col]
+].head()
+
+# +
+# create a column which indicates at least one outlier being found
+df_labeled_outliers["analysis.included_at_least_one_outlier"] = df_labeled_outliers[
+    [col for col in df_labeled_outliers.columns.tolist() if ".is_outlier" in col]
+].any(axis=1)
+
+# show value counts for all outliers
+outliers_counts = df_labeled_outliers[
+    "analysis.included_at_least_one_outlier"
+].value_counts()
+outliers_counts
+# -
+
+# show the percentage of total dataset
+print(
+    (outliers_counts.iloc[1] / outliers_counts.iloc[0]) * 100,
+    "%",
+    "of",
+    outliers_counts.iloc[0],
+    "include erroneous outliers of some kind.",
+)
+
+# show histograms to help visualize the data
+df_labeled_outliers.show_report()
+# +
+parquet_sampled_with_outliers = "./BR00117012_sampled_with_outliers.parquet"
+
+# check if we already have normalized data
+if not pathlib.Path(parquet_sampled_with_outliers).is_file():
+    # set a fraction for sampling
```

Comment:
might be helpful to add comment describing why 0.44

Reply:
Thanks, will do!